### PR TITLE
Fixes extremely thicc robots

### LINKED ISSUE #35712: Cyborg expand upgrade has no limit
https://i.imgur.com/fCgJGwG.png

Also you can reset the module of a cyborg and the expand upgrade will pop out of the borg without resetting its size so it has infinite uses

### PR DESCRIPTION
**Pull request by DaxDupont**
[Changelogs]: 

:cl: Dax Dupont
fix: After an incident where a very eager roboticist kept expanding a borg's size leading to a structural collapse of the entire station proper safety limitations have been implemented.
/:cl:

[why]: Fixes #35712

### CODE REVIEW

**Comment by deathride58** [comment]
ＥＸＰＡＮＤ　ＢＯＴ

**Comment by Bombpants** [comment]
How large is the largest size the robot can be?

**Comment by DaxDupont** [comment]
*2 as intended

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 1]
```diff
@@ -950,6 +951,10 @@
 	shown_robot_modules = FALSE
 	if(hud_used)
 		hud_used.update_robot_modules_display()
+
+	if (hasExpanded)
+		resize = 0.5
```
Comment (Cyberboss) [inline code/modules/mob/living/silicon/robot/robot.dm]: This won't actually do the resize, it's just a hook for admin varedits

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by DaxDupont** [comment]
Euh, since changing modules triggers update transform it works but I can
change it in a bit.

**Comment by DaxDupont** [comment]
Do you want me to copy paste the resizing code or just call update
transform

**Comment by Cyberboss** [comment]
just calling the proc there is fine

**Comment by DaxDupont** [comment]
@Cyberboss done

**Pull request merged**